scripts/screener/screener.runner.ts

```diff
@@ -53,6 +53,7 @@ async function scheduleScreenerBuild(
     },
     body: JSON.stringify({
       payload: payload,
+      isArtifactPresent: environment.screener.isArtifactPresent,
```

Comment:
This will pass a string i.e. `'true'` or `'false'`, correct? Is proxy aware that it will not be a `boolean`?

Reply:
Yes, both the proxy and CI are using this variable as string value

Reply:
is there a way to make it actual boolean ? within the payload and for proxy to consume it like that ? (I'm aware we are assign imperatively `'true'` `'false'` via ado pipeline config.)

Reply:
do we have interface that describes this payload somewhere ? if not can we create that ?

Reply:
the payload is described a few lines above and while there is a `ScreenerProxyPayload` interface, it seems to not be the same as this payload but I will create an additional PR that does that